package.json

```diff
@@ -93,6 +93,16 @@
                 "when": "editorTextFocus && vim.useCtrlKeys"
             },
             {
+                "key": "ctrl+a",
```

Comment:
I think both is right. It would be weird for one to work by default but not the other.